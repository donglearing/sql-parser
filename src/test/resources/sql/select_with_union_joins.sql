select t.a, f.b from T as t union join F as f union join H as h;