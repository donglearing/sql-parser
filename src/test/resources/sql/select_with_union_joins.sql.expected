SELECT t.a, f.b FROM T AS t UNION JOIN F AS f UNION JOIN H AS h